pkg/storage/db.go

```diff
@@ -542,7 +543,7 @@ with sel as (
            aws_resource_type.id as aws_resource_type_id,
            val.meta
     from (
-             values (text $1, text $2, text $3, text $4, jsonb $5)
+             values ($1::text, $2::text, $3::text, $4::text, $5::jsonb)
```

Comment:
In case this threw anyone off, `::` is used for type casting in postgres https://stackoverflow.com/a/15537761 I had to double check that lol